python/equistore-operations/equistore/operations/__init__.py

```diff
@@ -44,6 +44,12 @@
 )
 from .join import join
 from .lstsq import lstsq
+from .manipulate_labels_column import (
+    append_labels_column,
+    remove_labels_column,
+    rename_labels_column,
+    insert_labels_column,
```

Comment:
Naming-wise do we want `labels_column` or `_dimension`? Both are fine for me, with a slight preference for `_dimension`.


```suggestion
    append_dimension,
    remove_dimension,
    rename_dimension,
    insert_dimension,
```

Reply:
I also like the `_dimensions` a bit more. @jwa7  seems to be in favor of columns?

Reply:
these functions operate on TensorMap, the dimensions of which are always stored in Labels (each name/column in the Labels is a dimension of the high-dimensional sparse array we represent with a TensorMap). So for me the `_labels` part is redundant =)

Reply:
Fine for me, lets just wait if @jwa7 is also fine.

Reply:
@PicoCentauri @Luthaf Yep sounds good to me!